### PR TITLE
FINERACT-2120: Add new global configuration - enable payment hub integration.

### PR DESCRIPTION
**Pull request by kaibalya-fynarfin**
## Description

Describe the changes made and why they were made.

Ignore if these details are present on the associated [Apache Fineract JIRA ticket](https://github.com/apache/fineract/pull/1284).

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0147_use_payment_hub.xml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+    <changeSet author="fineract" id="1" context="postgresql">
+        <sql>
+            SELECT SETVAL('c_configuration_id_seq', COALESCE(MAX(id), 0)+1, false ) FROM c_configuration;
+        </sql>
+    </changeSet>
+    <changeSet author="fineract" id="2">
+        <insert tableName="c_configuration">
+            <column name="name" value="use-payment-hub"/>
```
Comment (fynmanoj) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/0147_use_payment_hub.xml]: "Enable payment hub integration " would be better name for this.

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0147_use_payment_hub.xml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            <column name="value"/>
+            <column name="date_value"/>
+            <column name="string_value" value="use payment hub"/>
+            <column name="enabled" valueBoolean="false"/>
```
Comment (fynmanoj) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/0147_use_payment_hub.xml]: Update tests and setups and teardowns for this global configuration

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0147_use_payment_hub.xml (hunk 1 above)
Comment (kaibalya-fynarfin) [reply fineract-provider/src/main/resources/db/changelog/tenant/parts/0147_use_payment_hub.xml]: i have updated the name of the global configuration.

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0147_use_payment_hub.xml (hunk 2 above)
Comment (kaibalya-fynarfin) [reply fineract-provider/src/main/resources/db/changelog/tenant/parts/0147_use_payment_hub.xml]: updated them and also added a new test for my newly added global configuration.

**Review by fynmanoj — APPROVED** [review]

**Comment by adamsaghy** [comment]
@kaibalya-fynarfin Kindly asking you to squash your commit! ;)

File: fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml [hunk 3]
```diff
@@ -165,4 +165,5 @@
     <include file="parts/0143_add_accrual_activity_posting_job.xml" relativeToChangelogFile="true" />
     <include file="parts/0144_transaction_summary_with_asset_owner_report_unc_allocation_fix.xml" relativeToChangelogFile="true" />
     <include file="parts/0145_job_short_name.xml" relativeToChangelogFile="true" />
+    <include file="parts/0147_enable_payment_hub_integration.xml" relativeToChangelogFile="true" />
```
Comment (fynmanoj) [inline fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml:168]: Can you rebase this file? Version 146 is missing

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml (hunk 3 above)
Comment (kaibalya-fynarfin) [reply fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml:168]: It is already rebased. version 146 is not the part of file changelog-tenant.xml. It is the part of final-changelog-tenant.xml

**Review by fynmanoj — APPROVED** [review]

**Review by fynmanoj — APPROVED** [review]

**Pull request merged**